Slider/Assets/Scripts/Sliders/Grid/SGrid.cs

```diff
@@ -410,6 +418,20 @@ public void GivePlayerTheCollectible(string name)
         }
     }
 
+    public void ActivateAllCollectibles(bool excludeSliders = false)
```

Comment:
I assume this is for debug? i would probably annotate it as such (and move to debug script instead of `SGrid.cs` if possible)